website/api/controllers/docs/view-basic-documentation.js

```diff
@@ -75,7 +75,8 @@ module.exports = {
         thisPage.meta.description ? thisPage.meta.description // « custom meta description for this page, if provided in markdown
         : 'Documentation for Fleet for osquery.'// « otherwise use the generic description
       ),
-      showSwagForm
+      showSwagForm,
+      docSearchPublicKey: sails.config.custom.docSearchPublicKey,
```

Comment:
I think this is used for searching more than the docs, right?

For consistency with `stripePublishableKey`, how about we call this `algoliaApiKey` or `algoliaPublishableKey` or `algoliaPublicKey`?  (Whatever you think is clearest)

**EDIT:** Ah, I see!  I am now reminded that Algolia calls this "docsearch".  This is not the first time I've forgotten this and then been surprised.

Nonetheless, for the sake of clarity, I think I'd still go with one of the above.  `algoliaDocSearchApiKey` etc would work too, but that just adds confusion, since folks like me who forgot the branding will wonder whether there is a different API key for handbook search, for articles search, etc.

Reply:
Makes sense! I've updated it to be `algoliaPublicKey`